### ISSUE TITLE
Python 2.6 is no longer supported by the Python core team, please upgrade your Python.

### ISSUE THREAD

**Issue by djspacedevil** [plesk/letsencrypt-plesk#70]
Hy,

i have installed the standard python for CentOS and Python2.7
But the module dont check this.

i have edit manuel the files for the last update... but this was overwrite on every update.

 And i use a different Port for the admin overview. 
Why is this not read dynamically from /etc/sw-cp-server/conf.d/plesk.conf??

```
Fehler: Let's Encrypt SSL certificate installation failed: Failed letsencrypt execution: /usr/local/psa/var/modules/letsencrypt/venv.bOCGa/lib/python2.6/site-packages/cryptography/__init__.py:25: DeprecationWarning: Python 2.6 is no longer supported by the Python core team, please upgrade your Python.
DeprecationWarning
2016-02-05 19:16:57,375:WARNING:letsencrypt.cli:Root (sudo) is required to run most of letsencrypt functionality.
An unexpected error occurred:
ConnectionError: HTTPSConnectionPool(host='127.0.0.1', port=8443): Max retries exceeded with url: /enterprise/control/agent.php (Caused by NewConnectionError('&lt;requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0x39e8f10&gt;: Failed to establish a new connection: [Errno 111] Connection refused',))
Please see the logfiles in /usr/local/psa/var/modules/letsencrypt/logs for more details. 
```

**Comment by xgin** [comment]
The binary packages are installed: cryptography and cffi.
It is built for every supported OS and its default python version - it is 13 different configurations!
The reason we do not use python27 if available: because we have to build binary packages for both versions.

There is a way to fix it: build own python and use it with letsencrypt.
I hope we will do it one fine day.

> Why is this not read dynamically from /etc/sw-cp-server/conf.d/plesk.conf??
It is, see #30 
Could you show the content of your plesk.conf

**Comment by dreiggy** [comment]
I have the same problem:

> Error: Let's Encrypt SSL certificate installation failed: Failed letsencrypt execution: /usr/local/psa/var/modules/letsencrypt/venv.xTSyE/lib/python2.6/site-packages/cryptography/__init__.py:25: DeprecationWarning: Python 2.6 is no longer supported by the Python core team, please upgrade your Python.
DeprecationWarning
2016-02-17 21:49:48,408:WARNING:letsencrypt.cli:Root (sudo) is required to run most of letsencrypt functionality.
An unexpected error occurred:
ConnectionError: HTTPSConnectionPool(host='127.0.0.1', port=8443): Max retries exceeded with url: /enterprise/control/agent.php (Caused by NewConnectionError('&lt;requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0x3a32710&gt;: Failed to establish a new connection: [Errno 110] Connection timed out',))
Please see the logfiles in /usr/local/psa/var/modules/letsencrypt/logs for more details.

**Comment by xgin** [comment]
The issue is fixed in the extension version 1.6.
The system python is no longer used, we have switched to plesk-py27 on all unix OSes.
